zulipterminal/ui_tools/views.py

```diff
@@ -778,6 +778,23 @@ def __init__(self, controller: Any, widgets: List[Any],
         self.log = urwid.SimpleFocusListWalker(widgets)
         super().__init__(self.log)
 
+    @staticmethod
+    def table_widths(contents: PopUpViewTableContent,
+                     dividechars: int=2,
+                     extra_width: int=0) -> Tuple[int, List[int]]:
```

Comment:
It isn't used anywhere for now.

I added for making it easier to add extra width if we ever want to. Should we remove it and add it only when it is needed?

Reply:
Let's leave it for now, unless we use it - actually, could it be used for padding the message info?

Reply:
It can only be used to increase the width from the inside - the content in the left column will be still left-aligned and the content in the right column will be still right-aligned. I will leave it for now.